sysutils/dokku-client/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            dokku dokku 0.29.0 v
+github.tarball_from     archive
+
+name                    dokku-client
+
+categories              sysutils
+
+maintainers             {@bashu gmail.com:bashu.was.here} openmaintainer
+
+description             Command-line client for the Dokku PaaS
+
+long_description        {*}${description}
+
+license                 MIT
+
+supported_archs         noarch
+
+checksums               rmd160  c1e871648a84a4341272b7f2f9f9e70a332e2673 \
+                        sha256  b7949af5a16981896dc01ddf0b372d9681304a94bd260f057928d69009c8cbc5 \
+                        size    2137220
+
+depends_run             path:bin/bash:bash
+
+use_configure           no
+
+build                   {}
+
+destroot {
+    xinstall -m 755 ${worksrcpath}/contrib/dokku_client.sh ${destroot}${prefix}/bin/dokku
```

Comment:
Perhaps the installed binary should also be renamed to `dokku_client`?

Reply:
Also make the permission `0755`

Reply:
> Also make the permission 0755

Fixed